packages/block-editor/src/components/rich-text/content.js

```diff
@@ -29,7 +29,7 @@ export const Content = ( { value, tagName: Tag, multiline, ...props } ) => {
 		value = `<${ MultilineTag }></${ MultilineTag }>`;
 	}
 
-	const content = <RawHTML>{ value }</RawHTML>;
+	const content = <RawHTML>{ value?.valueOf() }</RawHTML>;
```

Comment:
`toHTML` or `toHTMLString` (basically the existing rich text function lol)

Reply:
`toHTML` sounds great because HTML is already a string

Reply:
@dmsnell Wait till you hear about `HTMLElement` 😆

Reply:
But I do now remember why I used `valueOf`: it can be either a String or RichTextData, so this method is available in both cases. Same for `toString`. Maybe `toString` is good here?

Reply:
In any case, I will expose a method so it can be used if one wishes.

Reply:
Although I'm not opposed to doing an `instanceof` check

Reply:
Adjusted to `value instanceof RichTextData ? value.toHTMLString() : value`